domain/src/main/kotlin/no/nav/su/se/bakover/domain/behandling/Attestering.kt

```diff
@@ -0,0 +1,21 @@
+package no.nav.su.se.bakover.domain.behandling
+
+import no.nav.su.se.bakover.domain.NavIdentBruker
+
+data class Attestering(
+    val attestant: NavIdentBruker.Attestant,
+    val underkjennelse: Underkjennelse? = null,
```

Comment:
Kan vi unngå null ved å gjøre disse om til sealed classes f.eks? Mistenker at dette kanskje er en "snarvei" for å slippe å deale med forskjellen på yay/nay i databasen?

Reply:
Det var mest fordi det var den raskaste lösningen att implementera. Men enig i att f.eks sealed class är en bättre approach eftersom den är mer explicit.